### ISSUE TITLE
How can I encode control codes such as 'FUNC 1'

### ISSUE THREAD

**Issue by MugeSo** [Holger-Will/code-128-encoder#1]
I want to encode GS1-128. I requires 'FUNC 1' as the 1st code.

**Comment by Holger-Will** [comment]
func 1 is written as Ï. Just include it in your code as first character...

**Comment by MugeSo** [comment]
@Holger-Will Thanks.
I can insert FUNC 1, but encoded as a code128B.
How can I enforce as a code128C?

**Comment by Holger-Will** [comment]
True, func 1 is not treated as a digit by the optimizer. i will have a look into this.
